lib/matplotlib/backend_tools.py

```diff
@@ -903,26 +903,23 @@ def _mouse_move(self, event):
         self.navigation.canvas.draw_idle()
 
 
-tools = [['navigation', [(ToolHome, 'home'),
-                         (ToolBack, 'back'),
-                         (ToolForward, 'forward')]],
-
-         ['zoompan', [(ToolZoom, 'zoom'),
-                      (ToolPan, 'pan')]],
-
-         ['layout', [('ToolConfigureSubplots', 'subplots'), ]],
-
-         ['io', [('ToolSaveFigure', 'save'), ]],
-
-         [None, [(ToolGrid, 'grid'),
-                 (ToolFullScreen, 'fullscreen'),
-                 (ToolQuit, 'quit'),
-                 (ToolEnableAllNavigation, 'allnav'),
-                 (ToolEnableNavigation, 'nav'),
-                 (ToolXScale, 'xscale'),
-                 (ToolYScale, 'yscale'),
-                 (ToolCursorPosition, 'position'),
-                 (ToolViewsPositions, 'viewpos'),
-                 ('ToolSetCursor', 'cursor'),
-                 ('ToolRubberband', 'rubberband')]]]
+tools = [(ToolHome, 'home'), (ToolBack, 'back'), (ToolForward, 'forward'),
+         (ToolZoom, 'zoom'), (ToolPan, 'pan'),
+         ('ToolConfigureSubplots', 'subplots'),
+         ('ToolSaveFigure', 'save'),
+         (ToolGrid, 'grid'),
+         (ToolFullScreen, 'fullscreen'),
+         (ToolQuit, 'quit'),
+         (ToolEnableAllNavigation, 'allnav'),
+         (ToolEnableNavigation, 'nav'),
+         (ToolXScale, 'xscale'),
+         (ToolYScale, 'yscale'),
+         (ToolCursorPosition, 'position'),
+         (ToolViewsPositions, 'viewpos'),
+         ('ToolSetCursor', 'cursor'),
+         ('ToolRubberband', 'rubberband')]
+toolbar_tools = [['navigation', ['home', 'back', 'forward']],
```

Comment:
In the other PR... I put this one forward first to show my plans for getting rid of this altogether:
```python
if group is None:
  return
```
A convergence of multiple PRs working on the same bit of code to implement different new features ;).